pages/css/components/blankslate.md

```diff
@@ -77,18 +89,36 @@ Increases the size of the text in the blankslate
 
 ```html
 <div class="blankslate blankslate-large">
-  <h3>This is a blank slate</h3>
-  <p>Use it to provide information when no dynamic content exists.</p>
+  <img src="https://ghicons.github.com/assets/images/light/Pull%20Request.png" alt="" class="mb-3">
+  <h3 class="mb-1">You don’t seem to have any pull requests.</h3>
+  <p>Pull requests help you discuss potential changes before your changes are merged into the base branch.</p>
+  <button class="btn btn-primary my-3" type="button">New pull request</button>
+  <p><button class="btn-link" type="button">Learn more</button></p>
 </div>
 ```
 
-##### No background
+##### No border
+
+To remove the border, use the [border utliity](/css/utilities/borders) `.border-0`.
+
+```html
+<div class="blankslate border-0">
+  <img src="https://ghicons.github.com/assets/images/light/Pull%20Request.png" alt="" class="mb-3">
+  <h3 class="mb-1">You don’t seem to have any pull requests.</h3>
+  <p>Pull requests help you discuss potential changes before your changes are merged into the base branch.</p>
+  <button class="btn btn-primary my-3" type="button">New pull request</button>
+  <p><button class="btn-link" type="button">Learn more</button></p>
+</div>
+```
 
-Removes the `background-color` and `border`.
+Due to a design refresh of the blank slate component, we will be deprecating the use of `.blankslate-clean-background` which removes the `background-color` and `border`.
```

Comment:
Sorry one more thing! I think you could stick to just 
```suggestion
Due to a design refresh of the blank slate component, we will be deprecating the use of `.blankslate-clean-background`.
```

The additional info doesn't seem necessary and it's a little confusing whether or not you are saying `blankslate-clean-background` removes those rules or if the design refresh removes those rules. Maybe we could also add a warning emoji??